package.json

```diff
@@ -40,7 +40,10 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "chai-as-promised": "^5.2.0",
+    "fs-extra": "^0.26.7",
```

Comment:
It's used here: https://github.com/ember-fastboot/ember-fastboot-server/pull/39/files#diff-4b704b373a591b75fa93036c2f681941R95